### ISSUE TITLE
Block Inserter > Media > Generate images does not add alt text or title

### ISSUE THREAD

**Issue by jeffpaul** [10up/classifai#729]
### Describe the bug

When generating an image via the media modal, the prompt is added as the `Alternative Text` field and a hyphenated version is set for the `Title` field.  However when generating via the `Block Inserter > Media > Generate images` flow we're not properly putting the prompt into the `Alternative Text` field or setting that as the hyphenated version in the `Title` field.

### Steps to Reproduce

See above

### Screenshots, screen recording, code snippet

Block Inserter > Media > Generate images:
![Screenshot 2024-02-22 at 4 09 21 PM](https://github.com/10up/classifai/assets/2818133/6ba4a512-585a-47e4-b0a4-5d89949573f0)

Generated image in Media Library showing no `Alternative Text` and generic `Title`:
![Screenshot 2024-02-22 at 4 09 43 PM](https://github.com/10up/classifai/assets/2818133/ba043ee8-3f4e-422f-9182-830f788beccb)

### Environment information

Device: Mac mini 2018
OS: macOS Sonoma 14.3.1
Browser: Firefox 122.0.1

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Sidsector9** [comment]
@dkotter I couldn't find a way to fix this. Via the media modal, we set the file name which is used as the title and the alt text, but via the Block inserter > Generate images, the file name is set to `image.png` by core. This behaviour is consistent with Openverse as well.

https://github.com/10up/classifai/blob/77b529e47c0fec45244c48827df8f96931892471/src/js/gutenberg-plugins/inserter-media-category.js#L72-L77

The fields above are used when inserting the image block, but they are not submitted to the server-side when creating an attachment. Not sure if this is intentional by WP Core, do you suggest we raise a ticket?

**Comment by dkotter** [comment]
Yeah, I spent a little time looking at this when the issue was first opened and basically came to the same conclusion. Seems we're doing what we're supposed to here but Core isn't using the values we supply when the image is actually imported (it uses the title and alt text when rendering the preview image only).

I'd suggest seeing if there's an open issue in the Gutenberg repo (knowing it's not always easy finding things with the amount of issues there) and if there isn't an open one, we can open a new one.

**Comment by Sidsector9** [comment]
Opened an upstream feature request - https://github.com/WordPress/gutenberg/issues/60378
